src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -369,6 +386,36 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [isMobile]);
 
+    useEffect(() => {
+      if (isMobile) {
+        return;
+      }
+
+      const totalActiveDrawersMinSize = getActiveDrawersTotalMinSize();
+      const activeNavigationWidth = navigationOpen ? navigationWidth : 0;
+      // collapsed content width is $space-layout-content-horizontal * 2 = 48px
+      const minContentVisibleWidth = 48;
```

Comment:
There’s no way to calculate this directly from the DOM tree since `$space-layout-content-horizontal` is applied in the grid-template to `.` area, specifically in this line: https://github.com/cloudscape-design/components/blob/main/src/app-layout/visual-refresh-toolbar/skeleton/styles.scss#L56

Reply:
We already have this constant: https://github.com/cloudscape-design/components/blob/cc75463135eb8fa98a746cebdb6d3776673b934f/src/app-layout/visual-refresh-toolbar/compute-layout.ts#L32

I suggest to reuse it somehow